test/cases/ostree-ng.sh

```diff
@@ -84,7 +83,9 @@ COMPOSE_INFO=${TEMPDIR}/compose-info-${IMAGE_KEY}.json
 
 # SSH setup.
 SSH_OPTIONS=(-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5)
-SSH_KEY=${OSBUILD_COMPOSER_TEST_DATA}keyring/id_rsa
+SSH_DATA=$(/usr/libexec/osbuild-composer-test/gen-ssh.sh)
+SSH_KEY=${SSH_DATA}/id_rsa
+SSH_KEY_FILE=$(cat "${SSH_KEY}".pub)
```

Comment:
Swap variable names to actually match their contents. SSH_KEY_FILE is more appropriate for the file path, not for the file contents.